### PR TITLE
[Fixed] missing unlock and added a warning trace

### PR DESCRIPTION
**Pull request by matthiashanel**
Signed-off-by: Matthias Hanel <mh@synadia.com>

```
[4295] 2021/03/31 19:08:52.377093 [WRN] Account [AAODZMQUUPCOJW7RG7FJWR5FGZJE45GXHPUH45ZYFJGHMOQXMLBHPOQY] lookup for consumer create failed: fetching jwt timed out
```

### CODE REVIEW

File: server/jetstream_cluster.go [hunk 1]
```diff
@@ -2278,7 +2278,8 @@ func (js *jetStream) processConsumerAssignment(ca *consumerAssignment) {
 
 	acc, err := s.LookupAccount(ca.Client.serviceAccount())
 	if err != nil {
-		// TODO(dlc) - log error
+		js.mu.Unlock()
+		s.Warnf("Account lookup for consumer create failed: %v", err)
```
Comment (derekcollison) [inline server/jetstream_cluster.go:2282]: Should we put in which account we were trying to look up?

**Review by derekcollison — APPROVED** [review]
LGTM - Minor comment.

File: server/jetstream_cluster.go (hunk 1 above)
Comment (matthiashanel) [reply server/jetstream_cluster.go]: yes, fixed

**Review by derekcollison — APPROVED** [review]
LGTM - Thanks!

**Pull request merged**